static/app/views/insights/mobile/screens/components/screensOverview.tsx

```diff
@@ -0,0 +1,235 @@
+  if (searchQuery) {
+    query.addStringFilter(prepareQueryForLandingPage(searchQuery, false));
+  }
+  if (selectedPlatform) {
+    query.addFilterValue('os.name', selectedPlatform);
+  }
+  let queryString = query.formatString();
+
+  if (screens.length > 0) {
+    const screenFilter = `transaction:[${screens.map(name => `"${name}"`).join()}]`;
+    queryString = queryString + ' ' + screenFilter;
+  }
+
+  const newQuery: NewQuery = {
+    name: '',
+    fields: fields,
+    query: queryString,
+    dataset: dataset,
+    version: 2,
+    projects: selection.projects,
+  };
+  if (sortedBy) {
+    newQuery.orderby = sortedBy;
+  }
+  const tableEventView = EventView.fromNewQueryWithLocation(newQuery, location);
+
+  return {
+    eventView: tableEventView,
+    ...useTableQuery({
+      eventView: tableEventView,
+      enabled: enabled,
+      referrer: Referrer.SCREENS_SCREEN_TABLE,
+    }),
+  };
+}
+
+export function ScreensOverview() {
+  const router = useRouter();
+  const {selection} = usePageFilters();
+  const location = useLocation();
+  const organization = useOrganization();
+  const {isProjectCrossPlatform, selectedPlatform} = useCrossPlatformProject();
+  const [visibleScreens, setVisibleScreens] = useState<string[]>([]);
+  const sortedBy = decodeScalar(location.query.sort, '-count');
+  const sortField = decodeSorts([sortedBy])[0].field;
+
+  const transactionMetricsDataset = DiscoverDatasets.METRICS;
+  const transactionMetricsFields = [
+    SpanMetricsField.PROJECT_ID,
+    SpanMetricsField.TRANSACTION,
+    `count()`,
+    'avg(measurements.app_start_cold)',
+    'avg(measurements.app_start_warm)',
+    `avg(measurements.time_to_initial_display)`,
+    `avg(measurements.time_to_full_display)`,
+  ];
+
+  const spanMetricsDataset = DiscoverDatasets.SPANS_METRICS;
+  const spanMetricsFields = [
+    SpanMetricsField.PROJECT_ID,
+    SpanMetricsField.TRANSACTION,
+    `avg(mobile.slow_frames)`,
+    `avg(mobile.frozen_frames)`,
+    `avg(mobile.frames_delay)`,
+  ];
+
+  const isSpanPrimary = spanMetricsFields.some(
+    field => getAggregateAlias(field) === sortField
+  );
+  const primaryDataset = isSpanPrimary ? spanMetricsDataset : transactionMetricsDataset;
+  const primaryFields = isSpanPrimary ? spanMetricsFields : transactionMetricsFields;
+
+  const secondaryDataset = isSpanPrimary ? transactionMetricsDataset : spanMetricsDataset;
+  const secondaryFields = isSpanPrimary ? transactionMetricsFields : spanMetricsFields;
+
+  const {
+    data: primaryData,
+    isLoading: primaryLoading,
+    pageLinks: primaryLinks,
+    eventView: primaryEventView,
+  } = useMetrics(
+    primaryDataset,
+    primaryFields,
+    isProjectCrossPlatform ? selectedPlatform : undefined,
+    selection,
+    location,
+    sortedBy,
+    true,
+    []
+  );
+
+  const {
+    data: secondaryData,
+    // isLoading: secondaryLoading,
+    // pageLinks: secondaryLinks,
+    // eventView: secondaryEventView,
```

Comment:
Not sure if you wanted to keep these around for the future, but if not we could just remove them